en/guide/mavlink_2.md

```diff
@@ -80,19 +80,20 @@ if (mavlink_parse_char(MAVLINK_COMM_0, buf[i], &msg, &status)) {
 ```
 
 
-
 ## Message Extensions {#message_extensions}
 
-The XML for messages can contain extensions that are optional in the protocol. 
-This allows for extra fields to be added to a message.
+MAVLink 2 allows MAVLink 1 messages to be extended with *optional* fields.
+This allows for new fields to be added to a message without breaking binary compatibility for receivers that have not been updated.
 
-The rules for extended messages are:
+The rules for extensions messages are:
 
-* Extended fields are not sent in *MAVLink 1* messages. 
-* If received by an implementation that doesn't have the extended fields then the fields will not be seen.
-* If sent by an implementation that doesn't have the extended fields then the recipient will see zero values for the extended fields.
+* Extension fields are not sent in *MAVLink 1* messages. 
```

Comment:
This should probably be rephrased to "extensions are not present when a message is encoded as MAVLink 1", given that we send extensions on MAVLink 1 messages all the time. (I just want to ensure we don't confuse someone into thinking that you must have a message ID > 255 to use an extension)

Reply:
yes

Reply:
THanks! Changed to "Extension fields are not sent when a message is encoded using the *MAVLink 1* protocol. "